lib/ansible/modules/network/fortios/fortios_ipv4_policy.py

```diff
@@ -117,6 +117,15 @@
   application_list:
     description:
       - Specifies Application Control name.
+  logtraffic:
+    description:
+      - Logs sessions that matched policy.
+    default: utm
+    choices: ["disable", "utm", "all"]
+  logtraffic_start:
+      - Logs begining of session as well.
```

Comment:
Thanks for the quick reply!
I didn't pay attention to the 'please add version_added..' it should be corrected now.